### PR TITLE
Fixes #34252 - Missing example for 'Install packages' field

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

**Comment by stejskalleos** [comment]
[test foreman]

**Review by ezr-ondrej — COMMENTED** [review]
Why did we dropped the test? :thinking:
Not that it would be usefull much, but I do not see a reason to drop it in this PR (but I might be missing something).

**Comment by stejskalleos** [comment]
> Why did we dropped the test? thinking

I did same in https://github.com/theforeman/foreman/pull/9026, in past we had discussion about the snapshot tests and how they provide little to none value so I didn't bother to update them and deleted them right away.
I don't have any strong opinion here, no problem to revert it back and update the snapshots.

**Review by ezr-ondrej — APPROVED** [review]
The snapshot is pretty useless, but we should provide some tests instead of these.
It still has some value - if the component would have some hard errors, the snapshot would fail.
So it gives you "it rendered". Which is not great and far from enough value for a file with 20 lines of text, but at least something :)

But that's just a general thought, I'm ok dropping this one, because it is very simple component :+1:

File: app/controllers/api/v2/registration_commands_controller.rb [hunk 1]
```diff
@@ -17,7 +17,7 @@ class RegistrationCommandsController < V2::BaseController
         param :setup_remote_execution, :bool, desc: N_("Set 'host_registration_remote_execution' parameter for the host. If it is set to true, SSH keys will be installed on the host")
         param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours)")
         param :insecure, :bool, desc: N_("Enable insecure argument for the initial curl")
-        param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter. Example: `pkg1 pkg2`")
+        param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter, example: `pkg1 pkg2`.")
```
Comment (ofedoren) [inline app/controllers/api/v2/registration_commands_controller.rb:20]: Well, that's my mistake I didn't notice, but we avoid the dots at the end of param descriptions. At least for the sake of consistency.

**Comment by ezr-ondrej** [comment]
@stejskalleos can you address @ofedoren's concern in follow-up PR (with `Refs #34252..`) before we cherry-pick?

**Comment by ezr-ondrej** [comment]
CP 3.1: 5e3d9f76b

**Pull request merged**